.github/workflows/test.yml

```diff
@@ -17,6 +17,8 @@ jobs:
           check-latest: true
           cache: true
       - uses: golangci/golangci-lint-action@v6
+        with:
+          version: 1.60.1
```

Comment:
This is just to get a clean run with the linters currently having an issue in 1.60.2 and .3. We can wait until we remove this.